test/services/groups/transfer_service_test.rb

```diff
@@ -11,18 +11,43 @@ def setup
     end
 
```

Comment:
Can I get you to add a test for this  scenario

`Group A` -> member has `MAINTAINER` access
`Group A Subgroup A` -> above owner is added directly and member has `OWNER` access
`Group B` -> member has `GUEST` access

Transfer `Group B` under `Group A Subgroup A`

Ensure that in `Group A` the member still has `MAINTAINER` access and that only `Group A Subgroup A` and `Group B` has the member's access level been updated to `OWNER`

I would suggest adding all new groups, and members to the fixtures for this

Reply:
I changed `Group A` to `Group 9` and `Group B` to `Group 10`. Please see 43d8a91172044d228939828cc9effe16cf190e61.